Realm/RLMApp.h

```diff
@@ -175,3 +175,37 @@ to obtain a reference to an RLMApp.
 @end
 
 NS_ASSUME_NONNULL_END
+
+#pragma mark - Sign In With Apple Extension
```

Comment:
All of this could be in a separate file. See conventions with regards to categories in objc. Something like `RLMApp+ ASAuthorizationControllerDelegate.h`.

Reply:
Resolved offline